gwsurrogate/catalog.py

```diff
@@ -167,10 +167,23 @@
   (4,2), (5,5), (5,4), (5,3)]. The m<0 modes are deduced from the m>0 modes.
   Model details can be found in Rifat et al. 2019, arXiv:1910.10473. NOTE:
   the datasets in this hdf5 file are 32-bit (single) precision. Some are up-cast
-  to double in SurrogateIO.''',
+  to double in SurrogateIO. This model has been superseded by BHPTNRSur1dq1e4.''',
   '''https://arxiv.org/abs/1910.10473''',
   'd145958484738e0c7292e084a66a96fa')
 
+_surrogate_world['BHPTNRSur1dq1e4'] = \
+  surrogate_info(\
+  'https://zenodo.org/record/7125742/files/BHPTNRSur1dq1e4.h5',
+  '''Surrogate model 'BHPTNRSur1dq1e4' for non-spinning black hole binary
+  systems with mass-ratios varying from 2.5 to 10000. This surrogate model
+  is trained on waveform data generated by point-particle black hole
+  perturbation theory (ppBHPT), and tuned to NR simulations in the comparable 
+  (5,3),(5,4),(5,5),(6,4),(6,5),(6,6),(7,5),(7,6),(7,7),(8,6),(8,7),(8,8),
```

Comment:
Weird break in sentence after "comparable". Also, there is just a list of modes without referring to what that list is.